script.js

```diff
@@ -0,0 +1,27 @@
+// A function that adds and remove the class "active" on the section you click on.
+// We haven't really talked about ´this´ yet, but we will... ;)
+// console.log(this) to see how it works
+
+// Selects an HTML element, and calls a function which will be executed when the element is clicked.
+const accordionHeader1 = document.getElementById('accordion-header1');
+const accordionHeader2 = document.getElementById('accordion-header2');
+const accordionHeader3 = document.getElementById('accordion-header3');
```

Comment:
I like the choice of `document.getElementById(` 
When you can be specific, be specific 👍